grabbit/src/main/groovy/com/twcable/grabbit/GrabbitConfiguration.groovy

```diff
@@ -78,6 +79,23 @@ class GrabbitConfiguration {
         )
     }
 
+    private
+    static Collection<String> nonEmptyCollection(Collection<String> excludePaths, ConfigurationException.Builder errorBuilder) {
+        String prePattern = "^(/|\\./|\\|\\\\).*\$";
+        String postPattern = ".*(/|\\./|\\|\\\\)\$";
+        if (excludePaths == null || excludePaths.isEmpty())
+            return excludePaths
+        if (excludePaths.any({ String it -> (it == null) || (it.isEmpty()) })) {
```

Comment:
In Groovy, `it` is simply the name of the "unnamed" parameter. It's very weird to see it explicitly declared. (In fact, I was surprised that doing so wasn't a compile error.) Also, if the closure is the last (including "only") parameter to a function, it doesn't need to be inside method-call parens. Again, doing so is so unusual that it looks "really weird." :-)

This line can/should read simply
```groovy
if (excludePaths.any { it == null || it.isEmpty() }) {
```